pkg/discovery/network/weavenet.go

```diff
@@ -31,25 +31,19 @@ func discoverWeaveNetwork(clientSet kubernetes.Interface) (*ClusterNetwork, erro
 		return nil, err
 	}
 
-	var clusterNetwork *ClusterNetwork
+	clusterNetwork = &ClusterNetwork{
```

Comment:
Sorry, with my earlier change request I missed that this should now be

```suggestion
	clusterNetwork := &ClusterNetwork{
```